static/app/views/performance/traces/utils.tsx

```diff
@@ -9,3 +10,28 @@ export function normalizeTraces(traces: TraceResult<string>[] | undefined) {
     (t1, t2) => (t1.name ? '0' : '1').localeCompare(t2.name ? '0' : '1')
   );
 }
+
+export function getStylingSliceName(
+  sliceName: string | null,
+  sliceSecondaryName: string | null
+) {
+  if (sliceSecondaryName) {
+    return sliceSecondaryName.slice(-2) + (sliceName ? sliceName.slice(0, 3) : '');
```

Comment:
Could we leave a comment explaining why this is done this way? Without context, this is very cryptic.